src/unstract/adapters/vectordb/postgres/src/postgres.py

```diff
@@ -58,6 +59,9 @@ def get_vector_db_instance(self) -> BasePydanticVectorStore:
 
     def _get_vector_db_instance(self) -> BasePydanticVectorStore:
         try:
+            encrypted_password = urllib.parse.quote_plus(
```

Comment:
@harini-venkataraman I don't think url encoding is the better method to use for password encryption.

Reply:
@athul-rs This is not encryption. This is done for encoding special characters. 

Refer this for additional information: https://docs.sqlalchemy.org/en/20/core/engines.html#escaping-special-characters-such-as-signs-in-passwords

Reply:
@harini-venkataraman The variable naming needs to be changed here. It is confusing and misleading. Can you update it along with PR title?

Reply:
Changed. This was a wrongly named from my end.